gradle/libs.versions.toml

```diff
@@ -181,6 +182,7 @@ boms-micronaut-r2dbc = { module = "io.micronaut.r2dbc:micronaut-r2dbc-bom", vers
 boms-micronaut-flyway = { module = "io.micronaut.flyway:micronaut-flyway-bom", version.ref = "managed-micronaut-flyway" }
 
 boms-groovy = { module = "org.codehaus.groovy:groovy-bom", version.ref = "managed-groovy" }
+boms-grpc = { module = "io.grpc:grpc-bom", version.ref = "managed-grpc" }
```

Comment:
I think you don't need to reintroduce the `boms-grpc`, only the `managed-grpc` version, right?

Reply:
You're absolutely right, tested a snapshot and `micronaut-versions.properties` got correctly populated with `grpc.version=1.45.0` in starter 👍  thanks!

I'll get all this one day... 🤣